server/task_executor.go

```diff
@@ -159,7 +162,30 @@ func runMigration(ctx context.Context, l *zap.Logger, server *Server, task *api.
 	}
 
 	// If VCS based and schema path template is specified, then we will write back the latest schema file after migration.
-	if vcsPushEvent != nil && repository.SchemaPathTemplate != "" {
+	writeBack := (vcsPushEvent != nil) && (repository.SchemaPathTemplate != "")
+	// For tenant mode project, we will only write back latest schema file on the last task.
+	if issue != nil {
```

Comment:
only need to check this if writeBack is true

Reply:
Oh I see what you mean by performance, for loading issue.